### ISSUE TITLE
HTM comment file for image

### ISSUE THREAD

**Issue by Draky50110** [sebsauvage/MinigalNano#112]
Hello.

Trying to add a comment file for one imagE.
Name is ok but I put HTML language in it and... it is displayed as plain text.

So is it really HTML ?
What exactly can we put in image_filename.hml ?
What is "title:comment" tag ?

**Comment by tmos** [comment]
Hi @Draky50110 

TL;DR : you can't setup proper html, it's filtered.

But, this issue is just revelant of a bigger problem : we really need to have a better system for the pictures comments.

Today, we have a mix of what you use here : a file with `pic_name:comment`, and also `pic_name.ext.html`.

None of them is easily usable, and I think, not very functional.

We should get rid of those old features, and setup a new and easy to use way of binding data to pictures.

I think that a CSV-like file would be a nice option.

So every picture would be identified by its name, in the first col, and we could get every data we want, in the other columns.
We could setup some basic data (`id, title, description`), and let people use any data they want in their themes (`lat, long, people_name, date, place,...`). All of those informations could be used theme-side, without any modification of minigal code !

One CSV per folder could be a nice way of spliting data for performances.

What do you think about that ?

(ping to some people that may be interested on this issue : @nodiscc @Aldarone @sebsauvage )

**Comment by Draky50110** [comment]
Personnaly, I don't have any idea about this :p 
Was just trying to add comments for some pictures and never succeed.

**Comment by tmos** [comment]
No problem, I just used your issue to go deeper on the issue origin. I think you can consider the current state of comments as broken if it doesn't work as-is.

Will be fixed in a near future :)

**Comment by Draky50110** [comment]
Ok.
I keep following notifications in this issue ;)
Thanks :D

**Comment by nodiscc** [comment]
@tmos I think this is  good idea; I'm just unsure about CSV as a storgae format. IMHO the requirements for the comments file should be:
- 1 comment file per directory
- must be easily human-readable/writeable
- parsing the data should ideally rely on well known PHP libs (`php5-json`?)
